homeassistant/components/smappee/binary_sensor.py

```diff
@@ -68,7 +71,7 @@ def unique_id(
         return (
             f"{self._service_location.device_serial_number}-"
             f"{self._service_location.service_location_id}-"
-            f"presence"
+            f"{DEVICE_CLASS_PRESENCE}"
```

Comment:
```suggestion
            DEVICE_CLASS_PRESENCE
```

Reply:
Does not work for me, I get an error `unexpected token` which makes sense in my opinion since it does not know why the string suddenly stops and then a constant follows without any concatenation.

Reply:
Yeah.... the multiline f"" string is a bit odd here.

Reply:
The one line format didn't pass the commit tests